task_models/task_hmm.py

```diff
@@ -31,13 +31,13 @@ class HierarchicalTaskHMM(HierarchicalTask):
     _metaclass_ = ABCMeta
 
     def __init__(self, root=None, name=None, num_feats_action=None,
-                 share_feats=False, supp_bhvs=None):
+                 feats='cum', supp_bhvs=None):
```

Comment:
This is an ambiguous name. Why not using `cumulative` which is more self explanatory.

If I am not mistaken it appears from the following that `share_feats` and `feats` serve the same purpose. If so, please remove one.